map/style_tests/classificator_tests.cpp

```diff
@@ -301,9 +301,9 @@ UNIT_TEST(Classificator_PoiPriority)
   {
     CheckPriority({
       // 1
-      {"leisure", "playground"},
+      {"landuse", "recreation_ground"},
```

Comment:
Add new test and keep {"leisure", "playground"} here?

Reply:
`leisure=playground` is not supposed to have `sport=*`
its a playground for kids

Reply:
But still, there are a lot of them with sport=*
And leisure=sports_centre with sport=*

Reply:
There are 1230 playgrounds with sport=*
Can we clean them in the generator stage if it's obviously a bad mapping?

Reply:
`leisure-sports_centre` is being tested against `sport=` already

1230 playgrounds with sport=* is just 0.15% - a clear mismapping

Actually it would be good to "clean" many tags combinations we consider useless or weird or mismapping. E.g. recently there was a screenshot with a bicycle renting shop displaying its elevation (ele=*) in its PP..

Reply:
Cleaning the data is important to have a good-quality product. There will be always some data issues in OSM.

Reply:
- https://github.com/organicmaps/organicmaps/issues/6721